kano_updater/ui/install_window.py

```diff
@@ -45,6 +50,9 @@ def __init__(self):
         self.show_all()
         self._set_wait_cursor()
 
+        self._install_screen._pgl.hide()
```

Comment:
A bit cheeky to use private methods of objects outside the object but if there is no other way...